src/app/forms/templates/psv/psv-tech-record.template.ts

```diff
@@ -203,8 +203,49 @@ export const PsvTechRecord: FormNode = {
       label: 'Departmental vehicle marker',
       value: '',
 
+      type: FormNodeTypes.CONTROL,
+      viewType: FormNodeViewTypes.STRING
```

Comment:
This needs taken out of the vehicle summary and it's own smaller form node template creating. See the example I have merged into develop for how brakes work. The PR was https://github.com/dvsa/cvs-app-vtm/pull/315